xml/tuning_tuned.xml

```diff
@@ -0,0 +1,2501 @@
+            </para>
+          </listitem>
+        </varlistentry>
+        <varlistentry>
+          <term>powersave</term>
+          <listitem>
+            <para>
+              Optimizes the system for energy efficiency, potentially at the cost of performance.
+              While using this configuration as a stand-alone or merged profile, carefully analyze
+              the deployment use case and ensure that it saves more power than the
+              <literal>balanced</literal> profile.
+            </para>
+          </listitem>
+        </varlistentry>
+        <varlistentry>
+          <term>throughput-performance</term>
+          <listitem>
+            <para>
+              The <literal>throughput-performance</literal> profile maximizes overall system
+              performance for general-purpose servers handling diverse tasks. It sets the CPU
+              governor to performance mode, tunes I/O and network settings for high throughput, and
+              adjusts kernel parameters to enhance system performance.
+            </para>
+          </listitem>
+        </varlistentry>
+        <varlistentry>
+          <term>virtual-guest</term>
+          <listitem>
+            <para>
+              The <literal>virtual-guest</literal> profile optimizes performance and efficiency for
+              virtual machines running as guests. It tunes CPU and memory settings for virtualized
+              environments and adjusts disk and network settings to ensure optimal resource
+              utilization and VM performance.
+            </para>
+          </listitem>
+        </varlistentry>
+        <varlistentry>
+          <term>virtual-host</term>
+          <listitem>
+            <para>
+              The <literal>virtual-host</literal> profile enhances systems running KVM guests by
+              optimizing resource allocation and performance. It configures CPU and memory settings
+              for hosting multiple virtual machines, tunes I/O and network settings, and enhances
+              kernel parameters to support virtualization.
+            </para>
+          </listitem>
+        </varlistentry>
+      </variablelist>
+      <para>
+        For information on &tunedapp; profile configuration files, see the following sections:
+      </para>
+      <itemizedlist>
+        <listitem>
+          <para>
+            <xref linkend="sec-tuning-tuned-supported-profile-configuration"></xref>
+          </para>
+        </listitem>
+        <listitem>
+          <para>
+            <xref linkend="sec-tuning-custom-tuned-profile-configuration"></xref>
+          </para>
+        </listitem>
+      </itemizedlist>
+    </sect2>
+
+    <sect2 xml:id="sec--tuning-manage-tuned-profiles">
+      <title>Managing &tunedapp; profiles</title>
+      <para>
+        If you just want to activate a profile, pass the profile name as an argument to the
+        <option>--profile</option> while invoking the <command>tuned</command> command:
```

Comment:
In my humble opinion, explaining what options you need is a bit unnecessary and repetitive as you show it in the screen already. I think it's enough to shorten the sentence like this:

```suggestion
        If you want to activate a profile, pass the profile name to the <command>tuned</command> command:
```